### PR TITLE
added a method to validate and add conf dates

### PR DESCRIPTION
**Pull request by anushkrishnav**
Problem: Prevent the user from adding days from a different year

A way to handle this issue: #37

Conf.add_days will validate and add the dates

I can also add validation to check if the start date is not after the end date @Abdur-rahmaanJ

### CODE REVIEW

**Comment by anushkrishnav** [comment]
Not ready for review will work on the coverage a little later though I would like to discuss about the idea and like to know if you have any alternative approach

File: traveller/modules/conf/models.py [hunk 1]
```diff
@@ -24,6 +24,20 @@ class Conf(PkModel):
...
+        if int(cfp_start.year) != year:
+            raise ValueError("The year of the conference and the start date must be the same")
+
+        if int(cfp_end.year) != year:
+            raise ValueError("The year of the conference and the end date must be the same")
+
+        self.year = year
+        self.cfp_start = cfp_start
+        self.cfp_end = cfp_end
+
```
Comment (Abdur-rahmaanJ) [inline traveller/modules/conf/models.py:37]: Could you just trim those? Thanks!

**Comment by Abdur-rahmaanJ** [comment]
The validation is also a great idea! You can add it!

**Comment by anushkrishnav** [comment]
> The validation is also a great idea! You can add it!

will commit the changes soon

**Pull request closed without merging**